rollup.config.js

```diff
@@ -14,7 +14,7 @@ const convertImportToRequire = () => ({
 export default [
     {
 		external: ['kvstore', 'pubnub', 'xhr', 'crypto', 'codec/base64','codec/query_string', 'vault'],
-		input: 'src/functions/server.js',
+		input: 'src/server.js',
 		output: {
 			file: 'app/functions/server.js',
```

Comment:
Did we want the output file to remain in ```app/functions/server.js``` or ```app/server.js``?

Reply:
That's In the other PR